src/components/BoardHeader/HeaderMenu/HeaderMenu.tsx

```diff
@@ -18,7 +19,9 @@ type HeaderMenuProps = {
 const HeaderMenu = (props: HeaderMenuProps) => {
   const state = useSelector((applicationState: ApplicationState) => ({
     board: applicationState.board.data,
+    user: applicationState.users.all.find((user) => user.id === Parse.User.current()!.id),
```

Comment:
In theory, you could also get `showHiddenColumns` with `Parse.User.current().fetch()`

Reply:
Yes, there are several solutions -> e.g. `Parse.User.current()?.toJSON() as unknown as UserClientModel` should also work, but is not supported by our tests. I think we should leave it this way.